### ISSUE TITLE
Missing upload progress notification

### ISSUE THREAD

**Issue by lbdroid** [nextcloud/talk-android#3106]
### Steps to reproduce

After update to Talk 17 (F-Droid), there is no longer an upload progress notification.

Further, system settings --> app --> talk --> notifications now displays "this app does not send notifications".

There is no indication that the notification was replaced by anything (like in-conversation progress bar), which means that this functionality is now lost.

### Expected behaviour

There should be some indication of upload progress.

### Actual behaviour

There is none.

### Device brand and model

Pixel 6 Pro

### Android version

GrapheneOS 13

### Nextcloud Talk app version

17

### Custom Signaling server configured

None

### Custom TURN server configured

None

### Custom STUN server configured

None

**Comment by lbdroid** [comment]
Sweet, thank you very much!

**Comment by AndyXheli** [comment]
@mahibi dose this include all nofiticaion running on NC 27.0.0 Server Talk App 17.0.0 Androind Talk APP 17.0.0 and im not getting call or text notififacitons on the phone Samsung S22 Ultra Androind 13

**Comment by jurgenhaas** [comment]
Same problem here with Google Pixel 7 Pro and GrapheneOS 13. In the system settings, there are no permissions for notifications at all.

**Comment by mahibi** [comment]
> @mahibi dose this include all nofiticaion running on NC 27.0.0 Server Talk App 17.0.0 Androind Talk APP 17.0.0 and im not getting call or text notififacitons on the phone Samsung S22 Ultra Androind 13

this issue is about missing notification channels on GrapheneOS. 
notifications in general should work when using other Android OS with gplay services.

**Comment by lbdroid** [comment]
I've done a little more digging in this, and I've determined that this issue has NOTHING to do with GrapheneOS, and probably also nothing to do with Android 13.

I've reproduced the same problem on Nexus 6 running Lineageos 13.

Turns out that the issue has to do with the BUILD VARIANT. QA and GENERIC variants exhibit this problem. GPLAY variant does not.

Running a QA or GENERIC variant on EITHER device (Nexus 6 / Lineageos 13, Pixel 6 / GrapheneOS) results in failed upload progress notification.

Running the GPLAY variant on EITHER device results in WORKING upload progress notification. Note that the Pixel 6 / GrapheneOS does NOT have gservices installed.

One other observation (a difference between the lineageos and grapheneos) -- on the Nexus 6 Lineageos notification settings screen for GENERIC/Fdroid, it says "At your request, Android is blocking this app's notifications from appearing on this device."

Bottom line is that something in the QA/GENERIC variant is causing notifications to be disabled.

**Comment by ASerbinski** [comment]
Looks like POST_NOTIFICATIONS permission was getting added to the intermediate manifest for the gplay variant, but not qa or generic.
